src/langsmith/deploy-self-hosted-full-platform.mdx

```diff
@@ -75,7 +75,45 @@ As of v0.12.0, the `langgraphPlatform` option is deprecated. Use `config.deploym
       enabled: true
       langgraphPlatformLicenseKey: "YOUR_LANGGRAPH_PLATFORM_LICENSE_KEY"
 ```
-4. In your `values.yaml` file, configure the `hostBackendImage` and `operatorImage` options (if you need to mirror images)
+4. In your `values.yaml` file, configure the `hostBackendImage` and `operatorImage` options (if you need to mirror images). If you are using a private container registry that requires authentication, you must also configure `imagePullSecrets`:
+
+   <Accordion title="Configure authentication for private registries (optional)">
+
+   If your user deployments will use images from private container registries (e.g., AWS ECR, Azure ACR, private Docker registry), configure image pull secrets. This is a one-time infrastructure configuration that allows all deployments to automatically authenticate with your private registry.
```

Comment:
Is `user deployments` a specific terminology that we're using or is `user` meant to distinguish from the one-time platform deployment?

I'm fine with using `Agent Server deployment`. That's the new name I think.

Reply:
Ah yes, I've switched to agent server, much prefer, thanks